EXAMPLES.md

```diff
@@ -20,6 +20,7 @@
 | [Jens Getreu's blog](https://blog.getreu.net)                      |                                                          |
 | [Matthias Endler](https://endler.dev)                              | https://github.com/mre/mre.github.io                     |
 | [Michael Plotke](https://michael.plotke.me)                        | https://gitlab.com/bdjnk/michael                         |
+| [Mish Ushakov](https://mish.co)                                    |                                                          |
```

Comment:
It's not sorted so it's easier if you add it at the end of the list instead

Reply:
cool, will do! very proud to be using Zola for my website! i have a blog under /posts and projects under /projects, feel free to check out ;)